programs/anchor-movie-review-program/src/lib.rs

```diff
@@ -245,11 +245,31 @@ pub struct MovieAccountState {
     pub description: String,
 }
 
+/*
+    8 bytes for the anchor discriminator 
+    32 bytes for reviewer Pubkey 
+    1 bytes for the rating
+    4 bytes for the title String (still need to add String length in the account initialization)
+    4 bytes for the description String (still need to add String length in the account initialization),
+ */
+impl Space for MovieAccountState {
+    const INIT_SPACE: usize = 8 + 32 + 1 + 4 + 4;
```

Comment:
Same as other 2 PRs re: `#[derive(InitSpace)]` (for `MovieCommentCounter` below too).